presto-local-file/src/main/java/com/facebook/presto/localfile/LocalFileMetadata.java

```diff
@@ -150,12 +148,6 @@ public List<SchemaTableName> listViews(ConnectorSession session, String schemaNa
         return emptyList();
     }
 
-    @Override
```

Comment:
i think listViews above this is also duplicate.

Reply:
done